packages/devtools_app/lib/src/app.dart

```diff
@@ -215,7 +213,7 @@ class DevToolsAppState extends State<DevToolsApp> with AutoDisposeMixin {
                 .where((p) => embed && page != null ? p.screenId == page : true)
                 .where((p) => !hide.contains(p.screenId))
                 .toList();
-            if (tabs.isEmpty) return child;
+            if (tabs.isEmpty) return child ?? const SizedBox.shrink();
```

Comment:
we know child is not null because we specify it on line 236, so you can null assert child here